CHANGELOG.md

```diff
@@ -3,6 +3,21 @@
 This file lists the main changes with each version of the Fyne toolkit.
 More detailed release notes can be found on the [releases page](https://github.com/fyne-io/fyne/releases). 
 
+## 1.4 - Ongoing
+
+### Added
+
+*
+
+### Changed
+
+* Deprecate NewContainer in favour of NewContainerWithoutLayout
+
+### Fixed
+
+*
```

Comment:
I think this shouldn't be here right?

Reply:
I don't understand your point sorry. If you were meaning the fixed line isn't needed, this is just the standard template - it will get filled in quickly enough ;)

Reply:
Yeah I was just mentioning the little * character. I thought you accidentally added one :)